src/PageGenerator/project.json

```diff
@@ -5,18 +5,11 @@
     "url": "git://github.com/aspnet/diagnostics"
   },
   "dependencies": {
-    "Microsoft.AspNet.Diagnostics.Elm": "1.0.0-*",
-    "Microsoft.AspNet.Diagnostics.Entity": "7.0.0-*",
     "Microsoft.AspNet.Razor": "4.0.0-*"
   },
 
   "frameworks": {
-    "dnx451": { },
-    "dnxcore50": {
```

Comment:
We got a problem with assembly versioning on Razor's version of System.Runtime when we allowed CoreCLR. @pranavkm suggested we just remove it since this is a tool rather than a library.